spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'layouts/application', type: :view do
+  let(:mock_patron) { instance_double Patron, id: 'idhere' }
+
+  before do
+    without_partial_double_verification do
```

Comment:
this came from here: https://github.com/rspec/rspec-rails/issues/780#issuecomment-293111014

without it I couldn't mock this

I tried mocking ApplicationController#patron but rspec claimed that ApplicationController didn't implement `patron` 🤷